int32 INDEX
sensor_msgs/Image IMG
